README.md

```diff
@@ -10,7 +10,7 @@ Status: stage 2
 
 In WebIDL, possibly-ill-formed Strings are referred to using the [DOMString](https://webidl.spec.whatwg.org/#idl-DOMString) type. But for interfaces which operate on Unicode text, WebIDL also defines the [USVString](https://webidl.spec.whatwg.org/#idl-USVString) type as the set of all possible sequences of Unicode Scalar Values, which are all of the Unicode code points apart from the surrogate code points (`U+0000..U+D7FF` and `U+E000..U+10FFFF`), representing Unicode text.
 
-The WebAssembly [Component Model](https://github.com/WebAssembly/component-model) requires well-formed strings, as do some compile-to-JS programming languages, many data encodings, network interfaces, filesystem interfaces, etc. Interfacing JavaScript strings with such APIs is a common use case that therefore suffers from conversion burdens. In particular because conversion from `DOMString` to `USVString` is lossy (common options are to replace unpaired surrogates or to throw an error) there is a regular need for string validation both within the platform and for certain userland use case scenarios.
+The WebAssembly [Component Model](https://github.com/WebAssembly/component-model) has chosen to only support well-formed strings, even though it could have chosen to support `DOMString` as well. Unlike ECMAScript and similar Java-like languages, some other programming languages, many data encodings, network interfaces, filesystem interfaces, etc. support only well-formed strings. Interfacing JavaScript strings with such APIs is a common use case that therefore suffers from conversion burdens. In particular because conversion from `DOMString` to `USVString` is lossy (common options are to replace unpaired surrogates or to throw an error) there is a regular need for string validation both within the platform and for certain userland use case scenarios.
```

Comment:
Simplification
```suggestion
The WebAssembly [Component Model](https://github.com/WebAssembly/component-model) has chosen to only support well-formed strings, even though it could also support `DOMString`. Unlike ECMAScript and similar Java-like languages, some other programming languages, many data encodings, network interfaces, filesystem interfaces, etc. support only well-formed strings. Interfacing JavaScript strings with such APIs is a common use case that therefore suffers from conversion burdens. In particular, because conversion from `DOMString` to `USVString` is lossy (common options are to replace unpaired surrogates or to throw an error) there is a regular need for string validation both within the platform and for certain userland use case scenarios.
```